Source/DataModels/Components/UI/ComponentButton.h

```diff
@@ -22,19 +22,19 @@ class ComponentButton :
 	void SetClicked(bool clicked);
 	void SetHovered(bool hovered);
 
-	void SetColorClicked(float3 colorClicked);
-	void SetColorHovered(float3 colorHovered);
+	void SetColorClicked(const float4 colorClicked);
+	void SetColorHovered(const float4 colorHovered);
```

Comment:
```suggestion
	void SetColorHovered(const float4& colorHovered);
```

Same for the method above.